demo/src/main/java/com/novoda/accessibility/demo/custom_actions/TweetView.java

```diff
@@ -46,15 +45,38 @@ public void display(final String tweet, final Listener listener) {
 
         tweetTextView.setText(tweet);
 
+        if (services.isSpokenFeedbackEnabled()) {
+            setClickListenerToShowDialogFor(actions);
+        } else {
+            setIndividualClickListeners(tweet, listener);
+        }
+    }
+
+    private void setClickListenerToShowDialogFor(final Actions actions) {
+        setButtonsAsClickableFalseToFixErrorOnLollipopPlus();
+
+        setOnClickListener(
+                new OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        showAlertDialogFor(actions);
+                    }
+                }
+        );
+    }
+
+    private void setButtonsAsClickableFalseToFixErrorOnLollipopPlus() {
+        // https://code.google.com/p/android/issues/detail?id=205431
+        replyButton.setClickable(false);
```

Comment:
so gross.

Not sure whether to update this as the bug is "working as intended" or leave it to serve as a reasoning for why we do this. It's a super good description at least and helps explains some weird behaviour I've never understood on talkback.